tests/test_jams.py

```diff
@@ -101,8 +101,21 @@ def test_jobject_nonzero(data, value):
     assert J.__nonzero__() == value
 
 
-# Sandbox
+def test_jobject_repr():
+    repr(jams.JObject(foo=1, bar=2))
+
+
+def test_jobject_repr_html():
+    # Test once with empty
+    J2 = jams.JObject()
+    J2._repr_html_()
 
+    # And once with some nested values
+    J = jams.JObject(foo=1, bar=dict(baz=3), qux=[1], quux=None)
+    J._repr_html_()
```

Comment:
assert that this returns something? Ideally we'd want to validate that it's well-formatted HTML, but I couldn't easily (sub 30 seconds) find a library that does this?